tests/test_network.py

```diff
@@ -0,0 +1,33 @@
+from autowebcompat import network
+from keras import backend as K
+import math
+import numpy as np
+
+arr1 = np.array([[1, 1]], dtype=np.float32)
+arr2 = np.array([[0, 1]], dtype=np.float32)
+
+arr3 = np.array([[0, 0]], dtype=np.float32)
+arr4 = np.array([[3, 2]], dtype=np.float32)
+
+y_pred = np.array([0, 0.4, 0.7, 0.6, 0.5])
+y_true = np.array([0, 1, 0, 0, 1])
+
+
+def test_eucledian_distance():
+    dist = network.euclidean_distance([arr1, arr2])
+    assert (K.eval(dist) == 1)
+
+
+def test_eucl_distance_output_shape():
+    assert(network.eucl_dist_output_shape([arr1.shape, arr2.shape]) == (1, 1))
+    assert(network.eucl_dist_output_shape([arr3.shape, arr4.shape]) == (1, 1))
```

Comment:
Can you add a test with another array that has a different shape? Just to have a different output than just (1, 1).